Packs/FeedMISP/ReleaseNotes/1_0_38.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### MISP Feed
+
+- Fixed an issue where the **Max. indicators per fetch** parameter failing to halt the ***fetch-indicators*** process.
```

Comment:
need to rephrase this RN, why is a `Max. indicators per fetch` parameter responsible for halting fetch-indicators? I understand what you mean but think from a customer's perspective